.github/workflows/test-js.yml

```diff
@@ -0,0 +1,21 @@
+name: JS tests
```

Comment:
Would prefer a more obvious name, like "Frontend Javascript tests"

Reply:
My reason for making the name shorter is actually for the possibility of future README badges - I think it looks better to have a shorter, lowercase-oriented name

![JS tests](https://github.com/scpwiki/wikijump/workflows/JS%20tests/badge.svg)

Reply:
Hmm, okay